packages/common/core-interfaces/src/messages.ts

```diff
@@ -3,6 +3,19 @@
  * Licensed under the MIT License.
  */
 
+// Note: IRuntimeSignalEnvelope is an interface that mirrors ISignalEnvelope for signals that come from an external
+// caller (not sent by a client (so no 'clientBroadcastSignalSequenceNumber') and are always addressed
+// to the Container (so no 'address'):
+
+//  interface IRuntimeSignalEnvelope {
+// 		contents: {
+// 			type: string;
+// 			content: any;
+// 		};
+// 	}
+//
+// Make sure to reflect changes at 'server/routerlicious/packages/lambdas/src/utils/messageGenerator.ts'.
```

Comment:
I don't think we need a clone. You can put the reference in @privateRemarks in `ISignalEnvelope`'s comment